internal/wrappers/results-pdf-http.go

```diff
@@ -108,10 +109,11 @@ func (r *PdfHTTPWrapper) CheckPdfReportStatus(reportID string) (*PdfPollingRespo
 	}
 }
 
-func (r *PdfHTTPWrapper) DownloadPdfReport(reportID, targetFile string) error {
+func (r *PdfHTTPWrapper) DownloadPdfReport(url, targetFile string) error {
 	clientTimeout := uint(downloadTimeout)
-	customURL := fmt.Sprintf("%s/%s/download", r.path, reportID)
-	resp, err := SendHTTPRequest(http.MethodGet, customURL, http.NoBody, true, clientTimeout)
+	useAccessToken := FeatureFlags[MinioEnabled]
+	resp, err := SendHTTPRequestByFullURL(http.MethodGet, url, http.NoBody, useAccessToken, clientTimeout, "", true)
```

Comment:
The url includes the path?

Reply:
it's the url to the file on S3:
https://s3.dualstack.me-central-1.amazonaws.com/reports.uae.me-central-1-..........

Reply:
But it can be differnet one if MINIO false, no?

Reply:
casue it shoudn't be s3 in usual but checkmarx

Reply:
I'm approved for not being a blocker but please don't merge without verifying it